ui/app/AppMain.qml

```diff
@@ -195,15 +195,18 @@ RowLayout {
               anchors.topMargin: 0
 
               Rectangle {
+                  id: chatBadge
                   visible: chatsModel.unreadMessagesCount > 0
                   anchors.top: parent.top
                   anchors.left: parent.right
-                  anchors.leftMargin: -10
-                  anchors.topMargin: -5
-                  radius: width / 2
+                  anchors.leftMargin: -17
```

Comment:
Since this is positioned on the right, its way easier to use `anchors.right` and `anchors.rightMargin`

Reply:
> Since this is positioned on the right, its way easier to use `anchors.right` and `anchors.rightMargin`

```Chat Tap``` width might be changed dynamically according to unread messages count.
So, ```leftMargin``` is the proper solution in this case.
What do you think about it?